joplin/settings.py

```diff
@@ -151,6 +152,13 @@
     os.path.join(PROJECT_DIR, 'static'),
 ]
 
+WEBPACK_LOADER = {
+    'DEFAULT': {
+        'BUNDLE_DIR_NAME': 'webpack_bundles/',
+        'STATS_FILE': os.path.join(BASE_DIR, 'joplin/static/webpack-stats.json'),
```

Comment:
Make a note in the readme telling people to delete their webpack bundles folder if it gets too intense (also maybe make a techdebt issue to make this not a problem ever)